src/v2/Apps/FairOrginizer/FairOrganizerApp.tsx

```diff
@@ -14,10 +15,20 @@ interface FairOrganizerAppProps {
 const FairOrganizerApp: React.FC<FairOrganizerAppProps> = ({
   fairOrganizer,
 }) => {
-  const { name } = fairOrganizer
+  const { name, profile, slug } = fairOrganizer
+
+  const title = `${name} | Artsy`
+
   return (
     <>
-      <Title>{`${name} | Artsy`}</Title>
+      <MetaTags
+        description={`Search for fairs on ${name} on Artsy`}
```

Comment:
This might read a bit strangely, e.g. `Search for fairs on The Armory Show on Artsy`. Description is usually the longer-form text that has more details about the link being shared. Looks like there's an `about` [field for Fair Organizers](https://github.com/artsy/force/blob/3f64ac9d19cd841d5147f18653540124d3b385d1/src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerInfo.tsx#L40) - we should double-check that it works as description text, but I might start by querying for a few different fair organizer `about` fields and seeing how they look.

Reply:
Agree, sounds odd 😅 
The [Facebook recommendation](https://developers.facebook.com/docs/sharing/webmasters#basic) on it is around 2-4 sentences, [twitter recommends a maximum of 200 chars](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup), but in our case I can see long texts from the `about` field
```
1-54 Contemporary African Art Fair is delighted to return to Somerset House from Thursday 14 - Sunday 17 October 2021
for its 9th London edition. Since its inception, 1-54 has established itself as a leading voice in the global discussion on 
contemporary African art and has been at the forefront of the African art market with annual editions across three 
continents. As part of the programme of 1-54 Special Projects, the fair will once again be partnering with Christie’s to 
present an exhibition at the Duke Street space curated by art historian and art critic Christine Eyene and to deliver an online
 edition 1-54 Online Powered by Christie’s.\r\n\r\nFrom Thursday 14 – Sunday 31 October 2021, works from 1-54 will then be 
available to purchase and view on Artsy for an extended period of time following the close of the official show.\r\n\r\n1-54
 brings together a diverse set of perspectives from around the world and has carefully selected 47 leading international 
galleries from 19 countries across Europe, Africa and North America: Angola, Belgium, Brazil, Ivory Coast, Egypt, France, 
Germany, Ghana, Italy, Kenya, Morocco, Netherlands, Nigeria, Senegal, South Africa, Switzerland, Uganda, the United 
Kingdom and the United States.\r\n \r\nAmong the exhibitors this year are 20 galleries from Africa, a record number of 
galleries from the continent: Addis Fine Art, Afriart Gallery, Berman Contemporary, Circle Art Gallery, DADA Gallery, 
EBONY/CURATED, Galerie Atiss Dakar, Gallery 1957, Guns & Rain, Kalashnikovv Gallery, La Galerie 38, Loft Art Gallery, 
LouiSimone Guirandou Gallery, Mashrabia Gallery, Retro Africa, SMO Contemporary Art, THIS IS NOT A WHITE CUBE, THK 
Gallery, UBUNTU Gallery.
```